frontend/components/forms/fields/SelectTargetsDropdown/SelectTargetsInput/SelectTargetsInput.jsx

```diff
@@ -52,13 +52,17 @@ class SelectTargetsInput extends Component {
 
     const { handleInputChange } = this;
 
+    // must have unique key to select correctly
     const uniqueKeyTargets = targets.map((target) => ({
       ...target,
       unique_key: target.uuid || `${target.target_type}-${target.id}`,
```

Comment:
If you import `import { v4 as uuidv4 } from "uuid";` you can generate your own uuids on the fly: 

`uuid: uuidv4()`

Reply:
Also change `unique_key` to `uuid`.